.editorconfig

```diff
@@ -0,0 +1,21 @@
+root = true
+
+[*]
+
```

Comment:
Remove redundant space between the lines from here and down.

Reply:
I would also add the following under **[*]**
```
charset = utf-8
indent_size = 2
indent_style = space
tab_width = 4
insert_final_newline = true

[*.java]
indent_size = 4
```

Reply:
like this?
root = true

[*]

charset = utf-8
indent_size = 2
indent_style = space
tab_width = 4
insert_final_newline = true

[*.java]
indent_size = 4
trim_trailing_whitespace = true
ij_editorconfig_space_after_colon = false
ij_editorconfig_space_after_comma = true
ij_editorconfig_space_before_colon = false
ij_editorconfig_space_before_comma = false
ij_editorconfig_align_group_field_declarations = false
ij_editorconfig_spaces_around_assignment_operators = true
end_of_line = lf